### ISSUE TITLE
:preserved HAML filter throws rubocop Style/ColonMethodCall warnings

### ISSUE THREAD

**Issue by johnathanludwig** [brigade/haml-lint#74]
I have a few places where I use the `preserve` haml filter, and they get checked with rubocop and throw the ColonMethodCall warning.

![screenshot 2015-04-10 20 08 24](https://cloud.githubusercontent.com/assets/1262078/7099311/ae939194-dfbd-11e4-9fed-1e8d1fc2e6f7.png)

**Comment by sds** [comment]
Hey @johnathanludwig, thanks for the report.

The lint being reported here is not referring to the `:preserve` line, although I can understand the confusion. It's actually referring to some interpolation within the `:preserve` filter content itself, e.g.

```haml
:preserve
  #{Module::method}
```

You can fix the problem by writing:

```haml
:preserve
  #{Module.method}
```

Right now, all lints found in HAML filters are reported on the line of the filter's name. This obviously isn't ideal, but since we aren't heavy users of filters at Brigade we were comfortable with taking a shortcut here.

I've opened up #75 to track implementing a fix.

**Comment by johnathanludwig** [comment]
Well I feel dumb for not inspecting what was inside the preserve. I think a note in the readme regarding filters would be sufficient. I searched for information on them before posting, but couldn't find anything.

Thanks for the help, and great job on this linter!
